environments/se/core/src/main/java/org/jboss/weld/environment/se/logging/WeldSELogger.java

```diff
@@ -73,7 +73,7 @@
 
     // 2013 used in Weld 2.3
 
-    @Message(id = 2014, value = "Weld SE container {0} not initialized completely", format = Format.MESSAGE_FORMAT)
-    IllegalStateException weldContainerNotInitializedCompletely(Object id);
+    @Message(id = 2014, value = "Weld SE container {0} deployment not validated yet - methods for programmatic lookup cannot be used", format = Format.MESSAGE_FORMAT)
```

Comment:
Sounds a bit clumsy, how about a twist:
`Weld SE container with id {0} has not yet validated the deployment - methods ....`